pages/api/auth/[...nextauth].js

```diff
@@ -43,7 +43,7 @@ const options = {
       
       try {
         const res = await fetch(
-          `${process.env.API_ENDPOINT}/app/accountInfo`,
+          `${process.env.API_ENDPOINT}/treemapper/accountInfo`,
```

Comment:
The API name `treemapper/accountInfo` implies this API was created for the treemapper app. Why do we use it in here? It just doesn’t sound right 😄

Reply:
@norbertschuler the /app/accountInfo is not available in production/staging env yet.

Reply:
So it might be used for development purpose, but should be marked as experimental and never go to production this way.